lib/s3-sync.js

```diff
@@ -28,22 +28,22 @@ const jsonDocsDirDownloadOptions = {
 const jsonDocsDirUploadOptions = {
   localDir: 'tmp/json-docs',
   s3Params: {
-    ACL: 'public-read',
+    //ACL: 'public-read',
     Bucket: 'api-docs.emberjs.com',
     CacheControl: 'max-age=365000000, immutable',
     GrantRead: 'uri=http://acs.amazonaws.com/groups/global/AllUsers',
-    maxRetries: 5,
+    //maxRetries: 5,
     Prefix: 'json-docs'
   }
 }
 
 const revDocsDirUploadOptions = {
   localDir: 'tmp/rev-index',
   s3Params: {
-    ACL: 'public-read',
+    //ACL: 'public-read',
```

Comment:
ACL and maxRetries were causing errors with the version of aws client that the jsonapi-docs is using.